examples/oft-solana/tasks/solana/index.ts

```diff
@@ -107,3 +135,79 @@ export const getLayerZeroScanLink = (hash: string, isTestnet = false) =>
 
 export const getExplorerTxLink = (hash: string, isTestnet = false) =>
     `https://explorer.solana.com/tx/${hash}?cluster=${isTestnet ? 'devnet' : 'mainnet-beta'}`
+
+export const getAddressLookupTable = async (connection: Connection, umi: Umi, fromEid: EndpointId) => {
+    // Lookup Table Address and Priority Fee Calculation
+    const lookupTableAddress = LOOKUP_TABLE_ADDRESS[fromEid]
+    assert(lookupTableAddress != null, `No lookup table found for ${formatEid(fromEid)}`)
+    const addressLookupTableInput: AddressLookupTableInput = await fetchAddressLookupTable(umi, lookupTableAddress)
+    if (!addressLookupTableInput) {
+        throw new Error(`No address lookup table found for ${lookupTableAddress}`)
+    }
+    const { value: lookupTableAccount } = await connection.getAddressLookupTable(toWeb3JsPublicKey(lookupTableAddress))
+    if (!lookupTableAccount) {
+        throw new Error(`No address lookup table account found for ${lookupTableAddress}`)
+    }
+    return {
+        lookupTableAddress,
+        addressLookupTableInput,
+        lookupTableAccount,
+    }
+}
+
+export const getComputeUnitPriceAndLimit = async (
+    connection: Connection,
+    ixs: Instruction[],
+    wallet: KeypairSigner,
+    lookupTableAccount: AddressLookupTableAccount
+) => {
+    const { averageFeeExcludingZeros } = await getFee(connection)
```

Comment:
Error:

<img width="888" alt="image" src="https://github.com/user-attachments/assets/c8d02b4c-0482-4c83-b061-d2495f392ec7">

Seems running locally:
```
pnpm tsc
```

should catch this type of error

```
➜  oapp-prio-fees git:(master) ✗ pnpm hardhat lz:oft:solana:create --eid 40168 --program-id 3ZuKaovZkFLf5StLP2sHDBa65WKR7fjD1AViFugCFqRH --amount 77777777123 --only-oft-store true
bigint: Failed to load bindings, pure JS will be used (try npm run rebuild?)
No additional minters specified.  This will result in only the OFTStore being able to mint new tokens.
created SPL multisig @ APA9owkDi6SUUgXFbG5SaHb5nT8TpadA3mWK23DY6AzQ
Error fetching prioritization fees: TypeError: Cannot read properties of undefined (reading '_bn')
    at isPublicKeyData (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/@solana+web3.js@1.95.8/node_modules/@solana/web3.js/src/publickey.ts:39:35)
    at new PublicKey (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/@solana+web3.js@1.95.8/node_modules/@solana/web3.js/src/publickey.ts:58:9)
    at getPrioritizationFees (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/utils/getFee.ts:22:27)
    at getComputeUnitPriceAndLimit (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/index.ts:164:54)
    at addPerformanceInstructions (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/index.ts:195:81)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async SimpleTaskDefinition.action (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/createOFT.ts:231:29)
    at async Environment._runTaskDefinition (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/core/runtime-environment.ts:351:14)
    at async Environment.run (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/core/runtime-environment.ts:184:14)
    at async main (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/cli/cli.ts:324:7)
createTokenTx: https://explorer.solana.com/tx/4BuTsrPvAx1687uvFvvjZLHs9Erorwi5WnNwFbE4TMwAV3sDhRs4CfmH4Q6bkg4QDYTTKV9xqN2pERjsmFbXhBDG?cluster=devnet
Error fetching prioritization fees: TypeError: Cannot read properties of undefined (reading '_bn')
    at isPublicKeyData (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/@solana+web3.js@1.95.8/node_modules/@solana/web3.js/src/publickey.ts:39:35)
    at new PublicKey (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/@solana+web3.js@1.95.8/node_modules/@solana/web3.js/src/publickey.ts:58:9)
    at getPrioritizationFees (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/utils/getFee.ts:22:27)
    at getComputeUnitPriceAndLimit (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/index.ts:164:54)
    at addPerformanceInstructions (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/index.ts:195:81)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async SimpleTaskDefinition.action (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/createOFT.ts:262:25)
    at async Environment._runTaskDefinition (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/core/runtime-environment.ts:351:14)
    at async Environment.run (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/core/runtime-environment.ts:184:14)
    at async main (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/cli/cli.ts:324:7)
initOftTx: https://explorer.solana.com/tx/3Xs97mZUvXpq2V1VKiBAF7aaATgEXgDhxGXWbv21DXHw2J6uyre1z9SBZhmteHTf56TnJ8RuDR4V2GSpogSZsd4T?cluster=devnet
Error fetching prioritization fees: TypeError: Cannot read properties of undefined (reading '_bn')
    at isPublicKeyData (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/@solana+web3.js@1.95.8/node_modules/@solana/web3.js/src/publickey.ts:39:35)
    at new PublicKey (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/@solana+web3.js@1.95.8/node_modules/@solana/web3.js/src/publickey.ts:58:9)
    at getPrioritizationFees (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/utils/getFee.ts:22:27)
    at getComputeUnitPriceAndLimit (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/index.ts:164:54)
    at addPerformanceInstructions (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/index.ts:195:81)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async SimpleTaskDefinition.action (/Users/danielkmak/projects/oapps/oapp-prio-fees/tasks/solana/createOFT.ts:292:29)
    at async Environment._runTaskDefinition (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/core/runtime-environment.ts:351:14)
    at async Environment.run (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/core/runtime-environment.ts:184:14)
    at async main (/Users/danielkmak/projects/oapps/oapp-prio-fees/node_modules/.pnpm/hardhat@2.22.17_ts-node@10.9.2_typescript@5.7.2/node_modules/hardhat/src/internal/cli/cli.ts:324:7)
setAuthorityTx: https://explorer.solana.com/tx/5QtuXHuANz4XAQiDJRwaJmjtjkTLHyyTStrh7dzZ6g7RFNutbHqF554BxEgATenpoqL8rZNu2NH3B5hwcyVdyPB?cluster=devnet
Accounts have been saved to ./deployments/solana-testnet/OFT.json
```

Reply:
Got it, the internal API signature changed in #1084.  Will update it.

Reply:
this has been fixed by https://github.com/LayerZero-Labs/devtools/pull/1091/commits/8fbdfb139dbdb73b7f2feb33636e33bf05b397ff